jvb/src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -440,7 +529,28 @@ public void removeReceiver(String receiverId)
         }
     }
 
-    public interface EventHandler {
+    /**
+     * Notifies this instance that the specified receiver no longer wants or needs to receive anything from the media
+     * source attached to this instance (the sender).
+     *
+     * @param receiverId the id that specifies the receiver endpoint
+     * @param sourceName the media source name
+     */
+    public void removeReceiverV2(String receiverId, String sourceName)
+    {
+        ReceiverConstraintsMap sourceConstraints = receiverVideoConstraintsMapV2.get(sourceName);
+
+        if (sourceConstraints != null)
+        {
+            if (sourceConstraints.remove(receiverId) != null)
+            {
+                logger.debug(() -> "Removed receiver " + receiverId + " for " + sourceName);
+                receiverVideoConstraintsChanged(sourceConstraints.getMaxHeight());
```

Comment:
I think this needs to call `receiverVideoConstraintsChangedV2` with the source name

Reply:
yes, good catch